common/djangoapps/student/tests/test_verification_status.py

```diff
@@ -288,13 +288,11 @@ def _assert_course_verification_status(self, status):
         # Sanity check: verify that the course is on the page
         self.assertContains(response, unicode(self.course.id))
 
-        # Verify that the correct banner is rendered on the dashboard
-        self.assertContains(response, self.BANNER_ALT_MESSAGES[status])
-
```

Comment:
This has to be removed as you might have observe that we had remove banner alt messages due to accessibility issues.